source/apps/metadata.txt

```diff
@@ -0,0 +1,250 @@
+            view.
+
+            .. figure:: /images/select-an-organization.png
+               :alt: Menu showing a list of available Organizations
+               :width: 250px
+
+         .. step:: Find Your Project
+
+            Click :guilabel:`Projects` in the left navigation menu to view the
+            list of project in the Organization. Find your project in the
+            list.
+
+            .. figure:: /images/view-the-projects-list.png
+               :alt: The left navigation menu with the project selection highlighted
+               :width: 250px
+
+         .. step:: Copy the Project ID
+
+            The right side of the project list view contains an :guilabel:`Actions`
+            column. Click the ellipses icon to expand the Actions menu, and select
+            :guilabel:`Copy Project ID`.
+
+            .. figure:: /images/copy-the-project-id.png
+               :alt: Expanded ellipses icon showing the Actions menu and the copy
+               :width: 400px
+
+   .. tab:: Atlas CLI
+      :tabid: cli
+
+      You can list the Atlas projects that have access to with the Atlas
+      CLI's ``projects list`` command.
+
+      The command returns a list of projects in JSON format. Find
+      your project in the list and copy its ``id`` value.
+
+      .. io-code-block::
+
+         .. input::
+            :language: bash
+
+            atlas projects list
+
+         .. output::
+            :language: json
+            :emphasize-lines: 10
+
+            {
+              "links": [
+                {
+                  "rel": "self",
+                  "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups?pageNum=1\u0026itemsPerPage=100"
+                }
+              ],
+              "results": [
+                {
+                  "id": "5d0171d479328f10cb4f3037",
+                  "orgId": "629e04e11633f764462ea109",
+                  "name": "MyProject",
+                  "clusterCount": 1,
+                  "created": "2019-06-23T05:08:35Z",
+                  "links": [
+                    {
+                      "rel": "self",
+                      "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups/5d0171d479328f10cb4f3037"
+                    }
+                  ]
+                }
+              ],
+              "totalCount": 1
+            }
+
+   .. tab:: Atlas Admin API
+      :tabid: api
+
+      You can list the Atlas projects that have access to with the Atlas
+      Admin API's :atlas:`List All Projects
+      </reference/api-resources-spec/#tag/Projects/operation/listProjects>`
+      endpoint.
+
+      The endpoint returns a list of projects in JSON format. Find your
+      project in the list and copy its ``id`` value.
+
+      .. io-code-block::
+
+         .. input::
+            :language: bash
+
+            curl https://cloud.mongodb.com/api/atlas/v1.0/groups \
+              --digest -u <Atlas Public API Key>:<Atlas Private API Key>
```

Comment:
is this encoded in some way, or do you just pass plaintext keys? 

i ask b/c i remember having some issues using admin api w axios which doesn't do well w digest auth.

Reply:
cURL does the magic here - you give it your plaintext keys and it can do the rest. It can probably do it another way too but I'm not trying to get into the nitty gritty of digest auth here 😅

Reply:
ok, just checking. go cURL and do better axios!